.travis.yml

```diff
@@ -7,6 +7,9 @@ language: go
 go:
   - "1.11.x"
 
+env:
+  - GOFLAGS=-mod=vendor GO111MODULE=on
```

Comment:
`GO111MODULE=on` used to be configured on the Travis end apparently but was reverted to `auto` at some point:
https://github.com/terraform-providers/terraform-provider-helm/pull/186#issuecomment-462784515

Reply:
I can't actually find the moment in history when it was explicitly on in travis. It likely was forgotten in all postmodules changes.